src/editing.md

```diff
@@ -1,463 +1,489 @@
+Bitte beachte, dass Schlagwörter auf der Notiz-Ebene gespeichert werden. Das heißt,
+dass das Hinzufügen eines Schlagworts zu einer Karte automatisch auch
+alle Geschwister dieser Karte betrifft. Um eine einzelne Karte ohne ihre
+Geschwister zu kennzeichnen, kannst du Flaggen verwenden, da diese auf der
+Karten-Ebene gespeichert werden.
+
+### Verwendung von Flaggen
+
+Flaggen funktionieren ähnlich wie Schlagwörter, werden beim Wiederholen aber
+rechts oben angezeigt. Du kannst auch in der Kartenverwaltung nach Karten mit
+einer bestimmten Flagge suchen, Flaggen in der Seitenleiste umbenennen und Auswahlstapel
+aus Karten erstellen, die mit einer bestimmten Flagge gekennzeichnet wurden.
+Im Gegensatz zu Schlagwörtern kann eine Karte nur eine Flagge auf einmal zugewiesen bekommen. 
+Ein weiterer wichtiger Unterschied besteht darin, dass Flaggen auf der
+[Kartenebene](getting-started.md#karten) gespeichert werden, d. h. die Zuweisung
+einer Flagge auf eine Karte hat keine Auswirkungen auf deren Geschwister.
+
+Du kannst Karten direkt beim Wiederholen eine Flagge zuweisen bzw. diese wieder entfernen,
+indem du das Tastaturkürzel <kbd>Strg</kbd> + <kbd>1-7</kbd> (<kbd>⌘</kbd> + <kbd>1-7</kbd>
+auf dem Mac) drückst. Das funktioniert auch in der [Kartenverwaltung](browsing.md).
+
+
+### Markierung mit dem Schlagwort "marked"
+
+Anki behandelt das Schlagwort "marked" (engl. für "markiert") gesondert.
+Beim Wiederholen und in der Kartenverwaltung gibt es Funktionen, um
+dieses Schlagwort schnell hinzuzufügen oder zu entfernen.
+In der Wiederholungsansicht wird mit einem Stern in der linken oberen Ecke angezeigt,
+ob die aktuelle Karte das Schlagwort "marked" enthält. Karten, deren Notiz
+auf diese Weise markiert sind, werden in der Kartenverwaltung mit einer
+anderen Farbe hinterlegt.
+
+Info: Diese Funktion gibt es nur mehr aus Kompatibilitätsgründen mit älteren
+Versionen. In den meisten Fällen sind Flaggen besser geeignet.
+
+
+### Verwendung von Feldern
+
+Sollten Schlagworte nicht ausreichen, kann man auch designierte Felder zur
+inhaltlichen Klassifizierung verwenden, z. B. "Buch", "Seite" o. ä.
+Man kann die Suche nämlich auf bestimmte Felder einschränken. So sind sehr spezifische
+Suchanfragen möglich, wie z. B. `Buch:Grimm's Märchen Seite:63`.
+
+### Benutzerdefiniertes Lernen und Auswahlstapel
+
+Die Funktionen Benutzerdefiniertes Lernen und [Auswahlstapel](filtered-decks.md)
+ermöglichen dir, temporäre Stapel anhand von Suchbegriffen wie Schlagwörtern,
+Flaggen, Markierungen und Feldern zu erstellen. So kannst du im täglichen
+Gebrauch den Großteil deines Lernmaterials durchmischt in einem einzelnen
+Stapel wiederholen, bei Bedarf aber jederzeit temporäre Stapel erstellen, falls
+du einmal vollen Fokus auf ein bestimmtes Fach setzen musst, z. B. vor einer Prüfung.
+Im Allgemeinen gilt die folgende Regel: Wenn du einen Inhalt immer getrennt vom
+Rest wiederholen willst, solltest du ihn in ein normales Stapel geben;
```

Comment:
Yes indeed :smile: In my opinion, it would be for the best to change the application's translation to "Deck". It's a very commonly used word in german due to tabletop games etc. and everyone I know uses that term instead of "Stapel".